testonly/hammer/hammer.go

```diff
@@ -127,7 +127,13 @@ type MapConfig struct {
 	EPBias        MapBias
 	Operations    uint64
 	EmitInterval  time.Duration
-	IgnoreErrors  bool
+	IgnoreErrors, CheckSignatures bool
+}
+
+// String conforms with Stringer for MapConfig.
+func (c MapConfig) String() string {
+	return fmt.Sprintf("mapconfig:{mapID:%d biases:{%v} #operations:%d emit every:%v ignoreErrors? %t checkSignatures? %t",
```

Comment:
(nit) It feels like `Stringer` methods don't normally describe their own type, i.e. the "mapconfig:" bit might better belong in the caller.

Reply:
OK, done.